mappings/net/minecraft/world/gen/feature/StructureFeature.mapping

```diff
@@ -1,8 +1,67 @@
 CLASS net/minecraft/class_3195 net/minecraft/world/gen/feature/StructureFeature
+	FIELD field_38429 config Lnet/minecraft/class_3195$class_7302;
+	METHOD <init> (Lnet/minecraft/class_3195$class_7302;)V
+		ARG 1 config
+	METHOD method_38676 getStructurePosition (Lnet/minecraft/class_3195$class_7149;)Ljava/util/Optional;
+		ARG 1 positioner
 	METHOD method_38694 postPlace (Lnet/minecraft/class_5281;Lnet/minecraft/class_5138;Lnet/minecraft/class_2794;Ljava/util/Random;Lnet/minecraft/class_3341;Lnet/minecraft/class_1923;Lnet/minecraft/class_6624;)V
+		ARG 1 structureWorldAccess
+		ARG 2 structureAccessor
+		ARG 3 chunkGenerator
+		ARG 4 random
+		ARG 5 blockBox
+		ARG 6 chunkPos
+		ARG 7 structurePiecesList
 	METHOD method_41607 getValidBiomes ()Lnet/minecraft/class_6885;
 	METHOD method_41609 expandBoxIfShouldAdaptNoise (Lnet/minecraft/class_3341;)Lnet/minecraft/class_3341;
 		ARG 1 box
+	METHOD method_41612 getStructurePosition (Lnet/minecraft/class_3195$class_7149;Lnet/minecraft/class_2902$class_2903;Ljava/util/function/Consumer;)Ljava/util/Optional;
+		ARG 0 positioner
+		ARG 1 heightmapType
+		ARG 2 generator
+	METHOD method_41613 canGenerateAt (Lnet/minecraft/class_3195$class_7150;Lnet/minecraft/class_2794;Lnet/minecraft/class_7138;Ljava/util/function/Predicate;)Z
+		ARG 0 position
+		ARG 1 chunkGenerator
+		ARG 2 noiseConfig
+		ARG 3 biomePredicate
+	METHOD method_41614 getStructureStart (Lnet/minecraft/class_5455;Lnet/minecraft/class_2794;Lnet/minecraft/class_1966;Lnet/minecraft/class_7138;Lnet/minecraft/class_3485;JLnet/minecraft/class_1923;ILnet/minecraft/class_5539;Ljava/util/function/Predicate;)Lnet/minecraft/class_3449;
+		ARG 1 dynamicRegistryManager
+		ARG 2 chunkGenerator
+		ARG 3 biomeSource
+		ARG 4 noiseConfig
+		ARG 5 structureManager
+		ARG 6 seed
+		ARG 8 chunkPos
+		ARG 10 heightLimitView
```

Comment:
```suggestion
		ARG 10 world
```
(Technically `Chunk` is a `HeightLimitView` too; however as far as I'm aware any parameter taking World-implemented interface is named "world" for simplicity)